src/azul/service/manifest_service.py

```diff
@@ -652,11 +656,17 @@ def entity_type(self) -> str:
     def manifest_properties(cls, url: str) -> JSON:
         return {
             'command_line': {
-                'cmd.exe': None,
+                'cmd.exe': f'curl.exe {cls._process_url(url)} | curl.exe --config -',
                 'bash': f'curl {shlex.quote(url)} | curl --config -'
             }
         }
 
+    @classmethod
+    def _process_url(cls, url: str) -> str:
```

Comment:
```suggestion
    def _cmd_escape(cls, url: str) -> str:
    """
    Escape a URL for insert into a 'cmd.exe' command-line string.
```
Or other similar name to signify this function escapes a url (or string?) for use in a cmd.exe command-line string.

Reply:
For symmetry with shlex.quote I would call this method `cmd_exe_quote(s:str) -> str` 

Note the generic name of the argument.